docs/developer-docs/latest/setup-deployment-guides/configurations/required/middlewares.md

```diff
@@ -214,43 +393,136 @@ The middleware doesn't have any configuration options.
-| `frameguard`                | Set `X-Frame-Options` header to help mitigate clickjacking attacks<br/><br />Accepts the `action` parameter that specifies which directive to use.                                                                                                  | `String`                                                                     | `'sameorigin'`                                                             |
+| Option                      | Description                                                                                   | Type                  | Default value |
+|-----------------------------|-----------------------------------------------------------------------------------------------|-----------------------|---------------|
+| `crossOriginEmbedderPolicy` | Set the `Cross-Origin-Embedder-Policy` header to `require-corp`                               | `Boolean`             | `false`       |
+| `crossOriginOpenerPolicy`   | Set the `Cross-Origin-Opener-Policy` header                                                   | `Boolean`             | `false`       |
+| `crossOriginOpenerPolicy`   | Set the `Cross-Origin-Resource-Policy` header                                                 | `Boolean`             | `false`       |
+| `originAgentCluster`        | Set the `Origin-Agent-Cluster` header                                                         | `Boolean`             | `false`       |
+| `contentSecurityPolicy`     | Set the `Content-Security-Policy` header                                                      | `Boolean`             | `false`       |
+| `xssFilter`                 | Disable browsers' cross-site scripting filter by setting the `X-XSS-Protection` header to `0` | `Boolean`             | `false`       |
+| `hsts`                      | Set options for the HTTP Strict Transport Security (HSTS) policy.                             | `Object`              | See below     |
+| `hsts.maxAge`               | Number of seconds HSTS is in effect                                                           | `Integer`             | `31536000`    |
+| `hsts.includeSubDomains`    | Applies HSTS to all subdomains of the host                                                    | `Boolean`             | `true`        |
+| `frameguard`                | Set `X-Frame-Options` header to help mitigate clickjacking attacks, set to false to disable   | `Boolean` or `Object` | See below     |
+| `frameguard.action`         | Value must be either `deny` or `sameorigin`                                                   | `String`              | `sameorigin`  |
+
+::: tip
+When using any 3rd party upload provider, generally it's required to set a custom configuration here. Please see the provider documentation for which configuration options are required.
+:::
+
+::: strapi
```

Comment:
```suggestion
::: note
```

This should be a note callout :-) `strapi` callouts are for very specific use cases, such as highlighting some important concepts or explaining differences between two misunderstood topics, and then cross-linking to other sections of the documentation.